backend/tests/test_geoserver.py

```diff
@@ -0,0 +1,76 @@
+import os
+import pytest
+import asyncio
+from overstroomik_service.geoserver import Geoserver
+import warnings
+
+@pytest.mark.skipif(bool(os.getenv("GEOSERVER_HOST")) == False, reason="requires environment variable GEOSERVER_HOST (localhost)")
```

Comment:
Nice. When I get the compose integration working later I'll probably make this into something from `config.py`, then we don't need the cast, comparison and os.env here.

Reply:
use logging instead of warnings